library/sources/http-server/checkIfRequestIsBlocked.ts

```diff
@@ -88,6 +113,25 @@ export function checkIfRequestIsBlocked(
       ? agent.getConfig().isUserAgentBlocked(context.headers["user-agent"])
```

Comment:
Simplify line above by moving the new `userAgent` variable up to remove double checks?

Reply:
I did that initially but ended up leaves it to make the diff cleaner, maybe in a follow up PR we can refactor some things?

Reply:
https://github.com/AikidoSec/firewall-node/pull/601/commits/40a7f2e9845388c6bdba3badd283476cf3618a26